docs/src/Module.doc.js

```diff
@@ -36,6 +36,23 @@ card(
         type: 'string',
         required: true,
       },
+      {
+        name: 'expandedIndex',
+        type: '?number',
+        required: false,
+        description: [
+          'The 0-based index of the item that the expand/collapse state can be controlled programatically from an external component',
+        ],
+      },
+      {
+        name: 'onExpandedChange',
+        type: '(?number) => void',
+        required: false,
+        description: [
+          'The callback function that controls the expand/collapse state of an item if controlled programatically from an external component',
```

Comment:
"This callback is executed whenever any module item is expanded or collapsed. It receives the index of the currently expanded module, or `null` if none are expanded."